### DISCUSSION TITLE
Syslogs sent from Fortigate appliance not appearing on SO/Kibana Dashboards

### DISCUSSION THREAD

**Discussion by soad20000** [Security-Onion-Solutions/securityonion#11729] [Unsupported Versions]
Hello,

I have a standalone Security Onion 2.4.20 installation running in evaluation mode, meaning I can't ingest logs from an Elastic Agent. Right now I'm trying to visualize Syslog data sent from a Fortigate appliance but I can't seem to locate it. So far I've confirmed the following:

- Firewall rules on the SO node allow traffic from the Fortigate appliance on port 514 via TCP/UDP.
- Syslog data is being sent from the Fortigate appliance to the specified SO node ip address (same IP as the management interface), on port 514 via UDP.
- The SO node is receiving the Syslog data sent from the Fortigate appliance on the specified port. (Verified using tcpdump on the SO node).

However, I can't find any of the syslogs anywhere. I've tried filtering for syslog data in Kibana, and I've tried filtering for any data whose destination port is 514 but the logs come up empty. Could anyone point me in the right direction please? I can't install the Elastic Agent because SO is running in Eval mode. I don't understand why I can see the syslog traffic coming in on the management interface but I can't see it anywhere else.

Would greatly appreciate any info that could help me out.

**Comment by InfosecGoon** [comment]
Are there any errors in the Elasticsearch or Logstash logs? /opt/so/log/elasticsearch/ and /opt/so/log/logstash/, respectively.

If you do into Hunt and do a Log Type query from the built-in query menu, is there anything syslog related listed in the group metrics?

--Matt

**Reply by soad20000** [reply]
No idea what happened but a good old restart fixed it. The logs didn't give me any clues. I suspect it was an issue with the logs not being redirected to Kibana after being ingested. 

Also, it's worth mentioning that SO was running on a machine with only 16GB of RAM so that definitely didn't help. Since it was just a PoC what I ended up doing was spinning up a VM with the Elastic Stack on it and using an elastic agent to ingest the logs which was way easier thanks to the Fortigate integration.

Thanks to that we moved from PoC to production and I'm currently working on configuring a standalone deployment of SO, thankfully on a server with plenty of RAM and storage. Haven't had any issues yet, except for breaking sudo and having to do a complete reinstall (my fault) which was fun ;).

Thank you anyways Matt.
